Integrations/integration-fireeye.yml

```diff
@@ -2,8 +2,7 @@ commonfields:
   id: fireeye
   version: -1
 name: fireeye
-fromversion: 3.5.0
-display: FireEye (AX Series)
+display: gps_fireeye_integration
```

Comment:
Please return `display:` to be "FireEye (AX Series)"

Reply:
Updated display name to reflect default name